src/sourcemaputils.rs

```diff
@@ -278,3 +372,20 @@ impl SourceMapProcessor {
         Ok(())
     }
 }
+
+
+#[test]
+fn test_split_url() {
```

Comment:
Can we also add a test when a query is present ...
This is taken from a react generated file ...
`//# sourceMappingURL=/index.ios.map?platform=ios&dev=true&minify=false`